website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -646,6 +685,10 @@ The disks must be formatted & mounted in a RAID0 and be the underlying filesyste
 
 On AL2, Karpenter automatically configures the disks through an additional boostrap argument (`--local-disks raid0`). The device name is `/dev/md/0` and its mount point is `/mnt/k8s-disks/0`. You should ensure any additional disk setup does not interfere with these.
 
+#### AL2023
+
+On AL2023, Karpenter automatically configures the disks via the generated `NodeConfig` object. Like AL2, the device name is `/dev/md/0` and its mount point is `/mnt/k8s-disks/0`. You should ensure any additional disk setup does not interfere with these.
```

Comment:
Is this true now?

Reply:
Interesting, this is different from AL2 if this is the case. It's probably good that they are setting this default, but there is definitely a difference now in how this is handled.

Reply:
Never mind. Didn't realize that this was in the section that was specifically on RAID0. This looks good to me.